{"name": "test_login_alert[case0]", "status": "passed", "description": "异常用例，错误信息弹窗提示", "attachments": [{"name": "log", "source": "b6435d44-86db-4d25-aa42-a1ebe0fb3c58-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'mobile': '18684720550', 'pwd': 'python112', 'expected': '此账号没有经过授权，请联系管理员!'}"}], "start": 1590504059697, "stop": 1590504060615, "uuid": "6a38f23d-036f-41e7-bafd-2de6fa7d2d5c", "historyId": "228fe6f4a0f5b33a09dca3a5fc7b9e09", "testCaseId": "8b653efc132135221fc57baf686031ef", "fullName": "testcases.test_login.TestLogin#test_login_alert", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "wo"}, {"name": "thread", "value": "4740-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}